os/dbuild.sh

```diff
@@ -113,6 +113,7 @@ function SELECT_OPTION()
 				echo "  \"d. Download\""
 			fi
 			echo "  \"t. Build Test\""
+			echo "  \"l. Run Toolchain Command\""
```

Comment:
Can we add a small description/example for the user after `l` is entered ?

Reply:
Done. Thanks.